spade-core/src/main/scala/com/salesforce/mce/spade/orchard/WorkflowRequest.scala

```diff
@@ -73,7 +74,8 @@ object WorkflowRequest {
           r.name,
           r.resourceType,
           r.resourceSpec,
-          r.maxAttempt
+          r.maxAttempt,
+          r.terminateAfter.map(_.toHours.toDouble)
```

Comment:
let's make it have minute level precision instead. currently 1.5 hours would be converted to 1 hour.

Reply:
done